Frontend/src/components/Patients.jsx

```diff
@@ -1,167 +1,226 @@
-import React, { useState } from 'react';
-import { motion } from 'framer-motion';
-import { FaUserPlus, FaSearch, FaEdit, FaTrash } from 'react-icons/fa';
+import React, { useEffect, useState } from "react";
+import { motion } from "framer-motion";
+import { FaUserPlus, FaSearch, FaEdit, FaTrash } from "react-icons/fa";
+import axios from "axios";
 
 const Patients = () => {
-    const [patients, setPatients] = useState([
-        { id: 1, name: "John Doe", age: 35, gender: "Male", contact: "123-456-7890" },
-        { id: 2, name: "Jane Smith", age: 28, gender: "Female", contact: "987-654-3210" },
-    ]);
-    const [searchTerm, setSearchTerm] = useState('');
-    const [showForm, setShowForm] = useState(false);
-    const [newPatient, setNewPatient] = useState({ name: '', age: '', gender: '', contact: '' });
+  const [patients, setPatients] = useState([
+    {
+      id: 1,
+      name: "John Doe",
+      age: 35,
+      gender: "Male",
+      contact: "123-456-7890",
+    },
+    {
+      id: 2,
+      name: "Jane Smith",
+      age: 28,
+      gender: "Female",
+      contact: "987-654-3210",
+    },
+  ]);
+  const [searchTerm, setSearchTerm] = useState("");
```

Comment:
There will be a alert message when the patient name is not upto 4 characters. 
alert('Invalid name. Name should start with a letter and be at least 4 characters long.